Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM elixir:1.3.4
+RUN mkdir -p /app
+WORKDIR /app
+CMD ["iex", "-S", "mix"]
+COPY . /app
+RUN mix local.hex --force && \
+    mix deps.get && \
+    mix compile && \
+    mix credo --strict
```

Comment:
Ok, what I'd like to do is keep it until it throws one of these issues. Credo supports `@lint false` so we can explicitly turn off checks in specific functions. 

Though, I must say, I'm fine with that pipeline rule.